modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -198,10 +200,21 @@ public void processOpts() {
             setPackageVersion("1.0.0");
         }
 
+        if (additionalProperties.containsKey(CodegenConstants.ONLYPACKAGE_GENERATION)) {
+            setOnlyPackage(true);
```

Comment:
As this is a new feature, and is not overriding an inherited functionality. The pattern used for `excludeTests` would seem to be a better match. ie using a local method variable instead of having a class variable with getter/setter.